### DISCUSSION TITLE
科技爱好者周刊（第 256 期）：最酷的乐高作品 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#19] [General]
# 科技爱好者周刊（第 256 期）：最酷的乐高作品 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.plantree.me/weekly/issue-256

**Comment by hjg66-5** [comment]
没人评论？？？

**Reply by plantree** [reply]
不得不说 网友的脑洞是真大 能把乐高玩到这个水平.......
